src/client.ts

```diff
@@ -590,13 +590,9 @@ export interface IRequestMsisdnTokenResponse extends IRequestTokenResponse {
     intl_fmt: string;
 }
 
-interface IUploadKeysRequest {
+export interface IUploadKeysRequest {
```

Comment:
This type was incorrect, the usage here matches the new type [psf-874/test-messageactionbar](https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/crypto/index.ts#L2003-L2026)

Reply:
Added breaking change label

Reply:
I don't think this is a breaking change? The interface wasn't exported before, so it'd just be a regular bug fix.

Reply:
I wasn't really sure here so erred on the safe side. Removed the label.